src/lib.rs

```diff
@@ -552,7 +551,7 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
     pub fn resize(&mut self, new_len: usize, value: bool) {
         if new_len <= self.len {
             self.truncate(new_len);
-            return
+            return;
```

Comment:
This looks like it's a bug to me! Can we find a test case which triggers it?

Reply:
Pretty sure both `{ return }` and `return;` are type `()`

Reply:
Good point.